workflows/k8-calico-node/workflow.yaml

```diff
@@ -0,0 +1,148 @@
+    minavailable: 100%
+  # must complete in 1m
+  activeDeadlineSeconds: 86400
+  # keep workflows for 3m
+  ttlStrategy:
+    secondsAfterCompletion: 3600
+  # delete all pods as soon as they complete
+  podGC:
+    strategy: OnPodCompletion
+  serviceAccountName: argowf-svcacc
+  arguments:
+    parameters:
+      - name: appNamespace
+        value: "kube-system"
+      - name: appCurrentNamespace
+        value: "default"
+      - name: appLabel
+        value: "k8s-app=calico-node"
+      - name: appEndpoint
+        value: "localhost"
+      - name: fileName
+        value: "pod-custom-kill-health.json"
+      - name: chaosServiceAccount
+        value: chaos-admin
+  templates:
+    - name: argowf-chaos
+      steps:
+        - - name: pdbcreate
+            template: pdbcreate 
+        - - name: run-chaos
+            template: run-chaos
+        - - name: revert-chaos
+            template: revert-chaos
+
+    - name: pdbcreate
+      container:
+        image: alpine:latest
+        command: [sh, -c]
+        args: [sleep 10]
+
+    - name: run-chaos
+      inputs:
+        artifacts:
+        - name: run-chaos
+          path: /tmp/createChaosEngine.yaml
+          raw:
+            data: |
+              # chaosengine.yaml
+              apiVersion: litmuschaos.io/v1alpha1
+              kind: ChaosEngine
+              metadata:
+                name: k8-pod-delete
+                namespace: {{workflow.parameters.appCurrentNamespace}}
+              spec:
+                #ex. values: ns1:name=percona,ns2:run=nginx
+                appinfo:
+                  appns: {{workflow.parameters.appNamespace}}
+                  # FYI, To see app label, apply kubectl get pods --show-labels
+                  #applabel: "app=nginx"
+                  applabel: "app={{workflow.parameters.appLabel}}"
+                  appkind: deployment
+                jobCleanUpPolicy: delete
+                monitoring: false
+                annotationCheck: 'false'
+                engineState: 'active'
+                chaosServiceAccount: {{workflow.parameters.chaosServiceAccount}}
+                experiments:
+                  - name: k8-pod-delete
+                    spec:
+                      components:
+                        env:
+                          - name: NAME_SPACE
+                            value: {{workflow.parameters.appNamespace}}
+                          - name: LABEL_NAME
+                            value: {{workflow.parameters.appLabel}}
+                          - name: APP_ENDPOINT
+                            value: {{workflow.parameters.appEndpoint}}
+                          - name: FILE
+                            value: {{workflow.parameters.fileName}}
+                          - name: REPORT
+                            value: 'true'
+                          - name: REPORT_ENDPOINT
+                            value: 'https://eventbus-e2e.intuit.com/v2/ip-quality-reliability-realtime-prf'
```

Comment:
This line should be removed and should be parameterized {{workflow.parameters.reportEndpoint}}

Reply:
fixed